### PR TITLE
Fixes#1565 Adds Fossasia 2010 sample

### LINKED ISSUE #1565: Add Fossasia 2010 sample to the overview site.
Add Fossasia 2010 sample to the overview site. 

**Would you like to work on the issue?**

**Yes*

### PR DESCRIPTION
**Pull request by ParthS007**
Fixes #1565 

Changes: Add Fossasia 2010 sample to the overview site

Deployment link : [https://parths007.github.io/fossasia11/](https://parths007.github.io/fossasia11/)

### CODE REVIEW

**Comment by ParthS007** [comment]
@mariobehling  @geekyd  @agbilotia1998  Please review

File: test/serverTest.js [hunk 1]
```diff
@@ -440,7 +457,8 @@ describe('generate', function () {
       copyStatic('redhat.jpg');
       copyStatic('droidcon.jpg');
       copyStatic('fossasia16.jpg');
-      copyStatic('fossasia2011.jpg')
+      copyStatic('fossasia2011.jpg');
+      copyStatic('fossasia2010.JPG')
```
Comment (geekyd) [inline test/serverTest.js]: Add `;`

**Comment by ParthS007** [comment]
@geekyd  @agbilotia1998  Please review

**Comment by geekyd** [comment]
@ParthS007 @agbilotia1998 Let's add more features to the web-app, I think we have enough samples to showcase. Please have a look at issues related to adding features.

**Pull request merged**